extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -321,8 +317,8 @@ amp-story-page.i-amphtml-story-page-shown {
   opacity: 1!important;
 }
 
-.i-amphtml-story-share .i-amphtml-story-share-list > li + li {
-  margin-left: 8px!important;
+.i-amphtml-story-share .i-amphtml-story-share-list > li {
```

Comment:
can we remove rules on common tag names like `li`?

Reply:
Done.